3drepo/3drepo.io
# Issue 2706 - Added routing and skeleton for dashboard

This fixes #2706 

#### Description
- Added support for routing 
- Fully functional navigation for dashboard


#### Test cases
- Check that changing the selection boxes changes the url correctly.
- Check that selecting a tab changes the url correctly.
- Check that it doesnt reload the whole page when navigating.
- Check that refeshing matches the state of the selection boxes
- Check that foward and back button from the browser works correctly
